src/monarch_py/implementations/sql/sql_implementation.py

```diff
@@ -35,11 +35,13 @@ def get_entity(self, id: str, update: bool = False) -> Entity:
         Returns:
             Entity: Dataclass representing results of an entity search.
         """
-        
-        if not any(Path(monarchstow.base).iterdir()):
-            print("\nDownloading Monarch SQL KG...\n")
 
-        with monarchstow.ensure_open_sqlite_gz("sql", url=SQL_DATA_URL, force=update) as db:
+        with monarchstow.ensure_open_sqlite_gz(
+                "sql", 
+                url=SQL_DATA_URL, 
+                force=update, 
+                download_kwargs={'progress_bar':True}
```

Comment:
I made the progress bar default true so this line isn't necessary.

Reply:
Oh nice, that's perfect!! thanks so much for all your help on this, and getting it in so quickly!